Plugin.pm

```diff
@@ -288,6 +288,34 @@ sub trackInfoMenu {
 
 	my $search = cstring($client, 'SEARCH');
 	my $items = [];
+
+	my $artists = $track->remote ? $remoteMeta->{artists} : undef;
+	my $album_id = $track->remote ? $remoteMeta->{album_details}->{id} : undef;
+	my $track_id = Plugins::TIDAL::ProtocolHandler::_getId($track->url);
+
+	push @$items, {
+		name => $album,
+		line1 => $album,
+		line2 => $artist,
+		favorites_url => 'tidal://album:' . $album_id,
+		type => 'playlist',
+		url => \&getAlbum,
+		image => Plugins::TIDAL::API->getImageUrl($remoteMeta, 'usePlaceholder'),
+		passthrough => [{ id => $album_id }],
+	} if $album_id;
+
+	foreach my $_artist (@$artists) {
+		push @$items, _renderArtist($client, $_artist);
+	}
+
+	push @$items, {
+		name => cstring($client, 'PLUGIN_TIDAL_TRACK_MIX'),
+		type => 'playlist',
+		url => \&getTrackRadio,
+		image => 'plugins/TIDAL/html/mix_MTL_svg_stream.png',
```

Comment:
See above comment.

Reply:
I don't think the comment is placed correctly as I've not implemented the Track Radio thing here.  Do you mean the artist images?  Here I prefer the images, I think:

![image](https://github.com/michaelherger/lms-plugin-tidal/assets/8558/2f214590-d03e-44e5-8367-32a25d339dab)
![image](https://github.com/michaelherger/lms-plugin-tidal/assets/8558/2eb0ae6d-10a4-49de-b10f-34bf395f0ce4)
![image](https://github.com/michaelherger/lms-plugin-tidal/assets/8558/de07f5c8-aee4-49fe-ace5-098de610c7fd)
![image](https://github.com/michaelherger/lms-plugin-tidal/assets/8558/20c3bb88-fcbe-4ae6-bf59-7340d79fd630)